builder.opam

```diff
@@ -9,19 +9,18 @@ license: "ISC"
 depends: [
   "ocaml" {>= "4.08.0"}
   "dune" {>= "2.0.0"}
-  "asn1-combinators"
+  "asn1-combinators" {>= "0.3.0"}
   "bheap" {>= "2.0.0"}
   "bos"
   "cmdliner" {>= "1.1.0"}
-  "cstruct" {>= "6.0.0"}
   "duration"
   "fmt" {>= "0.8.7"}
   "fpath"
   "logs"
   "lwt"
   "ptime"
   "uuidm"
-  "http-lwt-client" {>= "0.2.0"}
```

Comment:
Hm what motivates this change?

Reply:
only 0.3.0 support tls 1.0.0 etc. (which allows us to use asn1-combinators 0.3.0), so I wanted to be clear that there's no expectation that an earlier one works with builder.